merge-files.js

```diff
@@ -122,26 +122,52 @@ async function main() {
   }
 
   // Detect project type
-  detectProjectType(sourceDir);
+  const projectType = detectProjectType(sourceDir);
+  console.log(`Detected project type: ${projectType}`);
+
+  if (projectType !== "unknown") {
+    const answer = await askQuestion(`We detected a ${projectType} project. Use default includes/excludes? (Y/N): `);
+    if (answer === "y" || answer === "yes") {
+      switch (projectType) {
+        case "node":
```

Comment:
@Monikarangasamy - Instead of this switch/case, maybe it is better to have a map like this:

```
projectFileMap = {
  "node": {
    "includes": [...],
    "excludes": [...]
  },
  ...
}
```